conmon/Makefile

```diff
@@ -2,7 +2,7 @@ src = $(wildcard *.c)
 obj = $(src:.c=.o)
 
 override LIBS += $(shell pkg-config --libs glib-2.0 json-glib-1.0)
-override CFLAGS += -std=c99 -Wall -Wextra $(shell pkg-config --cflags glib-2.0 json-glib-1.0)
+override CFLAGS += -std=c99 -Wall -Wextra $(shell pkg-config --cflags glib-2.0)
```

Comment:
There are currently some entries for the json-glib deps that can be removed also:
```
contrib/test/crio-integration-playbook.yaml
47:      - json-glib-devel
74:      - json-glib-devel

conmon/Makefile
4:override LIBS += $(shell pkg-config --libs glib-2.0 json-glib-1.0)
5:override CFLAGS += -std=c99 -Wall -Wextra $(shell pkg-config --cflags glib-2.0 json-glib-1.0)

conmon/conmon.c
21:#include <json-glib/json-glib.h>

Dockerfile
22:    libjson-glib-dev \

README.md
64:  json-glib-devel \
82:  libjson-glib-dev \
```
Absolutely not blocking, can be cleaned in a follow up.